.github/actions/drf-integrations-framework-build/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'DRF Integrations Framework Build'
+description: 'DRF Integrations Framework build steps'
+inputs:
+  python-version:
+    description: 'Python version to use'
+    required: true
+    default: '3.10'
+  django-version:
+    description: 'Django version to use'
+    required: true
+    default: '3.2'
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Python ${{ inputs.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ inputs.python-version }}
+    - name: Install Poetry
+      uses: snok/install-poetry@v1.3
+      with:
+        virtualenvs-create: true
+        virtualenvs-in-project: true
+        installer-parallel: true
+    - name: Load cached venv
+      id: cached-poetry-dependencies
+      uses: actions/cache@v2
+      with:
+        path: .venv
+        key: venv-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('**/poetry.lock') }}
+    - name: Install dependencies
+      if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+      run: poetry install --no-interaction --no-root
+      shell: bash
+    - name: Install Django version
+      run: python -m pip install "Django~=${{ inputs.django-version }}"
+      shell: bash
```

Comment:
Copied from https://github.com/yoyowallet/drf-react-template-framework/blob/master/.github/actions/drf-react-template-framework-build/action.yml - will move to a separate repo in the near future.